src/components/elements/CommonDetailGrid.tsx

```diff
@@ -0,0 +1,57 @@
+import { Grid, Typography } from '@mui/material';
+
+import React, { ReactNode } from 'react';
+
+// Grid of Key/Value pairs that has the specific styling used for Enrollment Details and custom client attributes
+
+export const CommonDetailGridContainer: React.FC<{ children: ReactNode }> = ({
+  children,
+}) => {
+  return (
+    <Grid
+      container
+      rowGap={0}
+      sx={{
+        '> .MuiGrid-item': {
+          borderBottomColor: 'borders.light',
+          borderBottomWidth: 1,
+          borderBottomStyle: 'solid',
+        },
+        '> .MuiGrid-item:nth-last-of-type(2)': {
+          border: 'unset',
+        },
+        '> .MuiGrid-item:nth-last-of-type(1)': {
+          border: 'unset',
+        },
+      }}
+    >
+      {children}
+    </Grid>
+  );
+};
+
+export const CommonDetailGridItem: React.FC<{
```

Comment:
This could probably could use some adjustment for accessibility. Maybe labeled by? Or[ aria table rows? ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/rowheader_role)

Reply:
Good call, thanks!

Reply:
Went with labelled-by since the grid isn't always a table, with a small enough screen it floats so the values are below the labels.
I also added aria labels to the edit buttons which were missing them